examples/passive_scalar/benchmark/cylinder2d/musubi.lua

```diff
@@ -0,0 +1,149 @@
+-- Diffusion process inside a 2D cylinder. The concentration profile is compared 
+-- to the analytical solution. In this simulation, anti-bounce back boundary 
+-- condition is used.
+
+require 'args'
+-------------------------------------------------------------------------------
+mesh               = './mesh/'  
+cs2 = 1./3.
+c_init = 1.
+maxN = 150.0
+tau = 0.516
+diff = (tau - 0.5) / 3
+tIni = 0
+
+function factorial(n)
+  if n < 0.1 and n > -0.1 then
+      return 1
+  else
+      return n * factorial(n - 1)
+  end
+end
+
+-- 0th and 1st Bessel function is computed for the concentration profile
+function bessel(param, order)
+  local sum = 0.
+  for iN = 0., maxN - 1 do
+    sum = sum + (-1)^iN / (factorial(iN) * factorial(iN + order)) * (
+      param / 2) ^ (2 * iN + order)
+  end
+  return sum
+end
+
+-- Analytical solution of the time evolution of the concentration profile
+function concentration_real(x, y, z, t)
+  local mu = {2.4048, 5.5201, 8.6537, 11.7915, 14.9309}
+  local sum = 0
+  local r = math.sqrt((x-nelem)^2 + (y-nelem)^2)
+  -- print("x = " .. x .. ", y = " .. y .. ", r = " .. r .. ", sum = ".. sum)
+  if (r > nelem) then
+    return cs2
+  end
+
+  for i = 1, #mu do
+    sum = sum + 2 / (mu[i] * bessel(mu[i], 1)) * math.exp(-mu[i] ^ 2
+      * diff * t / nelem ^ 2) * bessel(mu[i] * r / nelem, 0)
+    -- if x == 13 and y == 11 then  
+    --   print("mu[i] = " .. mu[i])
+    --   print("bessel(mu[i], 1) = " .. bessel(mu[i], 1))
+    -- end
+  end
+
+  local c = c_init * (1 - sum)
+  if c < 0.001 then
+    return 0.001 * cs2
+  end 
+  return c*cs2
+end
+
+-- Initial boundary condition can be assigned with analytical solution
+-- at a certain time step.
+function initial_concentration(x, y, z)
+  return concentration_real(x, y, z, tIni)
+end
+
+function concentration_afterIni(x, y, z, t)
+  return concentration_real(x, y, z, t + tIni) / cs2
+end
+
+-------------------------------------------------------------------------------
+sim_control        = {
+  time_control     = {
+    min      = { iter = 0        },
+    max      = { iter = t_total     },
+    interval = { iter = interval }
+  }
+}
+-------------------------------------------------------------------------------
+identify  = {
+  label = 'species', 
+  kind = 'passive_scalar', 
+  relaxation='trt', 
+  layout='d2q9',
+  order = 'second'
```

Comment:
``We have observed that there is need to extend this identify table to support several relaxations options for a scheme kind. So, we have fomulated a set of rules see #10. 

Here is the suggestion for this case
```
identify = {
  label = 'species',
  kind = 'passive_scalar',
  relaxation = {
    name = 'trt', 
    variant = 'second_order', 
  },
  layout = 'd2q9'
}
```

Also see #10 for how to name the compute routine.